manuscript/chapter3-contributor-mode.md

```diff
@@ -267,6 +273,7 @@ We saw that the first step is to teach to others but there also other steps or t
 * Read a lot of articles (every day or every few days): use Reddit, Dev.To, newsletters like Changelog/SoftwareleadWeekly
 * Study the evening before to sleep is very powerful because helps you to remember what you read more easily
 * Create a list of stuff to read or study when you have time
+* Read stuff from others that you admire
```

Comment:
Is the „that” here referring to „stuff” or do you mean „whom” (thereby referring to „others”)?